sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx

```diff
@@ -370,14 +370,16 @@ export const PlanPricingColumn = ({
             selectedInterval={selectedInterval}
             onIntervalChange={onIntervalChange}
           />
-          <TrialLink
-            planIds={planIds}
-            isUpgrade={isUpgrade}
-            planHasTrial={planHasTrial}
-            onButtonClick={checkoutTrial}
-            disabled={action?.disabled || isLoading}
-            dateFormat={shortDateFormat}
-          />
+          {allowAction ? (
```

Comment:
Aren't there any other UI changes like showing trail end date etc. on plans screen?

Reply:
it is already there https://github.com/raystack/frontier/blob/afb3116d7caaa3e4d21f01ca7b5d4c2646be4fd3/sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx#L203